bin/webdriver-manager

```diff
@@ -92,7 +106,9 @@ var cli = optimist.
     describe('ignore_ssl', 'Ignore SSL certificates').boolean('ignore_ssl').
     default('ignore_ssl', false).
     describe('proxy', 'Proxy to use for the install or update command').
-    describe('alternate_cdn', 'Alternate CDN to the binaries');
+    describe('alternate_cdn', 'Alternate CDN to the binaries').
+    describe('ie32', 'Use 32-bit IE driver regardless of current architecture').boolean('ie32').
```

Comment:
Maybe change the name of the binary to 'IEDriver-32bit', and remove the describe statement.